### PR TITLE
Updated code to work with python 3.4

### PR DESCRIPTION
**Pull request by hjgraca**
Updated code to work with python 3.4:

prints with parentheses 
urllib.parse
decode() on response.body

### CODE REVIEW

**Comment by hjgraca** [comment]
Maybe should have created a file specific to that version?!

File: src/update.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import tornado.options
 import json
 import sys
-import urlparse
+from urllib.parse import urlparse
```
Comment (oliver006) [inline src/update.py]: This breaks for Python2.7 but you change it to 

```
try:
    from urllib.parse import urlparse
except:
    from urlparse import urlparse
```

and then it should work for both 2.7 and Python 3

**Comment by hjgraca** [comment]
Uses different urlparser depending on python version

File: src/update.py [hunk 2]
```diff
@@ -17,21 +20,21 @@ def download_and_index_item(item_id):
     url = "https://hacker-news.firebaseio.com/v0/item/%s.json?print=pretty" % item_id
     h = {'User-Agent': "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/37.0.2062.124 Safari/537.36"}
     response = yield http_client.fetch(url, headers=h)
-    item = json.loads(response.body)
+    item = json.loads(response.body.decode())
```
Comment (oliver006) [inline src/update.py]: I think this needs to be `decode('utf-8')` or otherwise it'll fail for eg id=8837508 (failed for me)

**Pull request merged**